lib/diagnostic.rb

```diff
@@ -13,7 +13,15 @@
 # and location should only be writable.
 
 ##
-# your code here
+class Person
+  attr_accesor :name
+  attr_reader :age
+  def initialize (name, age, location)
+    @name = name
+    @age = age
+    @location =  location
+  end
+end
```

Comment:
Correct but don't forget to add Location to a attr. In the question, it asked for it to be writable but not readable. So it would be a `attr_writer`